components/org.wso2.carbon.deployment.engine/src/main/java/org/wso2/carbon/deployment/engine/config/DeploymentConfiguration.java

```diff
@@ -38,7 +38,7 @@
     private String runtimeRepositoryLocation = "${wso2.runtime.path}/deployment/";
 
     @Element(description = "Scheduler update interval")
-    private int updateInterval = 15;
+    private int updateInterval = 5;
```

Comment:
Why are we updating here without doing it at the deployment.yml level?

Reply:
This is to make the behaviour default, thus we need not add the configurations. PArt of discussions for, https://groups.google.com/d/msg/siddhi-dev/j3W4mA8jjQ8/rRKeF6isBAAJ